Exercises.String/Exercises.String/GoogleMapsModel.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Exercises.String
+{
+    class GoogleMapsModel
+    {
+        public Dictionary<string, List<string>> UrlStruct { get; }=new Dictionary<string, List<string>>();
+
+
+        public GoogleMapsModel(string url)
+        {
+            string _url = DecodeUrl(url);
+            int i = 0;
+            while (i < _url.Length)
+            {
+                StringBuilder currentValue = new StringBuilder();
+                StringBuilder currentKey = null;
+                List<string> currentValues = new List<string>();
+
+                if (_url[i] == '&')
```

Comment:
Please move string or chars literals to constants declared at the top of the class